smart_contract.sol

```diff
@@ -144,94 +74,98 @@ contract ForkDelta is SafeMath {
     admin = admin_;
   }
 
-  function changeFeeAccount(address feeAccount_) {
-    if (msg.sender != admin) revert();
+  function changeFeeAccount(address feeAccount_) public isAdmin {
     feeAccount = feeAccount_;
   }
 
-  function changeFeeMake(uint feeMake_) {
-    if (msg.sender != admin) revert();
-    if (feeMake_ > feeMake) revert();
+  function changeFeeMake(uint feeMake_) public isAdmin {
+    require (feeMake_ <= feeMake);
     feeMake = feeMake_;
   }
 
-  function changeFeeTake(uint feeTake_) {
-    if (msg.sender != admin) revert();
-    if (feeTake_ > feeTake) revert();
+  function changeFeeTake(uint feeTake_) public isAdmin {
+    require(feeTake_ <= feeTake);
     feeTake = feeTake_;
   }
 
-  function changeFreeUntilDate(uint freeUntilDate_) {
-    if (msg.sender != admin) revert();
+  function changeFreeUntilDate(uint freeUntilDate_) public isAdmin {
     freeUntilDate = freeUntilDate_;
   }
 
-  function deposit() payable {
+  function deposit() public payable {
     tokens[0][msg.sender] = safeAdd(tokens[0][msg.sender], msg.value);
     Deposit(0, msg.sender, msg.value, tokens[0][msg.sender]);
   }
 
-  function withdraw(uint amount) {
-    if (tokens[0][msg.sender] < amount) revert();
+  function withdraw(uint amount) public {
+    require(tokens[0][msg.sender] >= amount);
     tokens[0][msg.sender] = safeSub(tokens[0][msg.sender], amount);
-    if (!msg.sender.call.value(amount)()) revert();
+    require(msg.sender.call.value(amount)());
     Withdraw(0, msg.sender, amount, tokens[0][msg.sender]);
   }
 
-  function depositToken(address token, uint amount) {
+  function depositToken(address token, uint amount) public {
     //remember to call Token(address).approve(this, amount) or this contract will not be able to do the transfer on your behalf.
-    if (token==0) revert();
-    if (!Token(token).transferFrom(msg.sender, this, amount)) revert();
+    require(token!=0);
+    require(Token(token).transferFrom(msg.sender, this, amount));
     tokens[token][msg.sender] = safeAdd(tokens[token][msg.sender], amount);
     Deposit(token, msg.sender, amount, tokens[token][msg.sender]);
   }
 
-  function withdrawToken(address token, uint amount) {
-    if (token==0) revert();
-    if (tokens[token][msg.sender] < amount) revert();
+  function withdrawToken(address token, uint amount) public {
+    require(token!=0);
+    require(tokens[token][msg.sender] >= amount);
     tokens[token][msg.sender] = safeSub(tokens[token][msg.sender], amount);
-    if (!Token(token).transfer(msg.sender, amount)) revert();
+    require(Token(token).transfer(msg.sender, amount));
     Withdraw(token, msg.sender, amount, tokens[token][msg.sender]);
   }
 
   //support ERC223
-  function tokenFallback( address _origin, uint _value, bytes _data) public returns (bool ok){return true;}
+  function tokenFallback( address _origin, uint _value, bytes _data) public returns (bool ok)
+  {
+      return true;
```

Comment:
Let's move the { on 147 up to the end of 146. Just to keep consistency.

Reply:
Sure.